sources/models/src/lib.rs

```diff
@@ -134,6 +134,8 @@ struct KubernetesSettings {
     allowed_unsafe_sysctls: Vec<SingleLineString>,
     server_tls_bootstrap: bool,
     cloud_provider: KubernetesCloudProvider,
+    registry_qps: u32,
+    registry_burst: u32,
```

Comment:
In the kubelet reference page, these are both documented as taking an `int32`, which corresponds to Rust's `i32` type.

Reply:
Negative numbers are valid values?

Reply:
I expect they're not valid, but neither is the upper half of the unsigned 32-bit range.